assets/js/modules/analytics/components/dashboard/DashboardBounceRateWidget.js

```diff
@@ -147,5 +148,6 @@ function DashboardBounceRateWidget() {
 
 export default whenActive( {
 	moduleName: 'analytics',
-	fallbackComponent: AnalyticsInactiveCTA,
+	FallbackComponent: <AnalyticsInactiveCTA />,
+	IncompleteComponent: <CompleteModuleActivationCTA slug="analytics" />,
```

Comment:
See other comments, this needs to then become
```suggestion
	FallbackComponent: AnalyticsInactiveCTA,
	IncompleteComponent: () => <CompleteModuleActivationCTA slug="analytics" />,
```

Reply:
I've updated this and each of the 8 other iterations of this prop.